backbone.js

```diff
@@ -381,6 +381,8 @@
   // want global "pubsub" in a convenient place.
   _.extend(Backbone, Events);
 
+  var defaultIdAttribute = 'id';
```

Comment:
If exposed, could be an easy, one-time configuration for couchDB and mongoDB users.